python/ct/crypto/in_memory_merkle_tree.py

```diff
@@ -0,0 +1,125 @@
+"""In-memory Merkle Tree
+
+Operates (and owns) an in-memory array of leaves which can be updated.
+Not particularly efficient.
+"""
+
+import math
+
+from ct.crypto import merkle
+
+def _down_to_power_of_two(n):
+    """Returns the power-of-2 closest to n"""
```

Comment:
C6108: 12: One-line docstring summary should end with ".", "?", or "!" [g-short-docstring-punctuation]